.husky/commit-msg

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1 --color
```

Comment:
Same here, is this code generated by husky? In my opinion commitlint should be added as a devDependency. In this case we do not need npx afaik.

Reply:
Like the other files you've commented on, these were generated by husky so I haven't touched them (other than adding the color arg). It seems that this is the standard way of doing things (https://github.com/typicode/husky/blob/main/.husky/commit-msg) but this is my first time using husky so I'm not certain on this.

Reply:
Ah ok, thx for the clarification.